Formula/php.rb

```diff
@@ -61,6 +60,7 @@ class Php < Formula
   uses_from_macos "libxml2"
   uses_from_macos "libxslt"
   uses_from_macos "zlib"
+  uses_from_macos "libffi", since: :catalina
```

Comment:
This probably needs to be after `libedit` but before `libx*`. The audit will complain if needed, though.

Reply:
Fixed.